AsyncDisplayKit/ASVideoNode.h

```diff
@@ -15,7 +15,7 @@
 // in an issue on GitHub: https://github.com/facebook/AsyncDisplayKit/issues
 
 @interface ASVideoNode : ASControlNode
-@property (atomic, strong, readwrite) AVAsset *asset;
+@property (atomic, strong, readonly) AVAsset *asset;
```

Comment:
@gazreese this is technically a breaking API change. It may be the right thing to do, and now is the time to change it if we are going to. What was your thinking around this part of the change, And do you feel it is important or beneficial?

cc @lappp9, @Xinchi thoughts?